jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -7182,6 +7182,128 @@
 						"description": ""
 					},
 					"response": []
+				}
+			]
+		},
+		{
+			"name": "u) health check application test",
+			"description": "",
+			"item": [
+				{
+					"name": "start jvm with wait true",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "179a918a-9d12-4d3a-9594-17818fefa228",
+								"type": "text/javascript",
+								"exec": [
+									"setTimeout(function(){}, 5000);",
```

Comment:
Do we still need this timeout in the test if the call to start the JVM is using setting synchronous flag to true?